### PR TITLE
Nemtseva Olga test task

### PR DESCRIPTION
**Pull request by olga-nemtseva-quantori**
Positive and negative tests for send CV form

### CODE REVIEW

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 1]
```diff
@@ -0,0 +1,126 @@
+﻿Feature: 3.1. Send CV
+
+Background: 
+	Given User is on the Job description page of the Carrier Portal
+	And the Send CV form is visible
+	Example:https://career.quantori.com/ru/positions/cloud-python-engineer
```
Comment (romanLiovi) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: Example: https://career.quantori.com/ru/positions/cloud-python-engineer
You should leave a space after keyword "Example". Moreover, it is better to use keyword "Examples", because the word that you used is a synonim to the word "Scenario", so Given, When, and Then steps are needed

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+		|132342 678 |digits, two words    |
+
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_05 Fill the field "Email" with incorrect data
+	When user enters "<sample data>" (<comment>) value into the "Email" field
+	And user clicks on another form field to move the focus
+	Then a red exclamation mark (!) appears at the right side of the "Email" field
+	Examples:
+		|sample data |comment                  |
```
Comment (romanLiovi) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: Just to add: domain should consist of two or more words and numbers in it are excluded

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+@Priority:Critical
+Scenario Outline: 3.1_08 User did not attach CV file
+	Given user filled all form with correct data
+	When user did not attach CV file
+	And user pressed "Send" button
+	Then text on the "Attach CV" button should become red
+	And the form is not sent, e.g. "Thanks for being awesome!" message does not appear
+
+@Type:Negative
+@Priority:Critical
```
Comment (romanLiovi) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: I'm not sure that priority is critical.

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+Scenario Outline: 3.1_05 Fill the field "Email" with incorrect data
+	When user enters "<sample data>" (<comment>) value into the "Email" field
+	And user clicks on another form field to move the focus
+	Then a red exclamation mark (!) appears at the right side of the "Email" field
+	Examples:
+		|sample data |comment                  |
+		|            |empty field              |
+		|            |one or more spaces       |
+		|@.com       |empty username and domain|
+		|вася@mail.ru|with non-English symbols |
```
Comment (romanLiovi) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: Non-English symbols are allowed, but not in domain

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 5]
```diff
@@ -0,0 +1,126 @@
+﻿Feature: 3.1. Send CV
+
+Background: 
+	Given User is on the Job description page of the Carrier Portal
+	And the Send CV form is visible
```
Comment (romanLiovi) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:5]: Maybe you should also add that CV form is clear

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+		|RTF   |.rtf     |
+	
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_04 Fill the field "Name" with incorrect data
+	When user enters "<sample data>" (<comment>) value into the "Name" field
+	And user clicks on another form field to move the focus
+	Then a red exclamation mark (!) appears at the right side of the "Name" field
+	Examples:
+		|sample data|comment              |
```
Comment (romanLiovi) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: You can also check two or more words with such symbols as "@", "=" and etc. For examle: Olg@ N"

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+	And the form is not sent, e.g. "Thanks for being awesome!" message does not appear
+
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_08 User attached an empty CV file
+	Given user filled all form with correct data
+	When user attached a CV file with 0 bytes size
+	And user pressed "Send" button
+	Then text on the "Attach CV" button should become red
+	And the form is not sent, e.g. "Thanks for being awesome!" message does not appear
```
Comment (romanLiovi) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:136]: In addition, great job! I found your homework much better than mine)

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+@Priority:Critical
+Scenario Outline: 3.1_07 Attach incorrect CV file
+	Given user filled all form with correct data
+	When user attached CV file with the following parameters: <description>
+	And user pressed "Send" button
+	Then a message should appear: "Something went wrong"
+	Examples:
+		|description                                  |
+		|file with .jpeg extension - wrong file format|
+		|file with .pdf extension, >10 Mb size        |
```
Comment (romanLiovi) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:118]: You can also try there boundary value analysis

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature (hunk 1 above)
Comment (olga-nemtseva-quantori) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: Ok, will fix

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature (hunk 3 above)
Comment (olga-nemtseva-quantori) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: Yes, agree. At first I supposed it's the same as not attached CV

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature (hunk 4 above)
Comment (olga-nemtseva-quantori) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: will fix

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature (hunk 5 above)
Comment (olga-nemtseva-quantori) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:5]: Yes, I will add it

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature (hunk 6 above)
Comment (olga-nemtseva-quantori) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature]: Yes, I agree and thanks for reminding about SQL  injections)

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature (hunk 7 above)
Comment (olga-nemtseva-quantori) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:136]: Thank you)

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature (hunk 8 above)
Comment (olga-nemtseva-quantori) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:118]: Thanks. Added this to `Scenario Outline: 3.1_03 Attach correct CV file` scenario because it's a positive CV file test scenario.

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 9]
```diff
@@ -0,0 +1,136 @@
+﻿Feature: 3.1. Send CV
+
+Background: 
+	Given User is on the Job description page of the Carrier Portal
```
Comment (yankovskayaksenia) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:4]: Better to provide links for sites and buttons , so how we can get to that page, like:
 Given google.com is opened
And user clicked on "" button
And..

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+	And the Send CV form is visible
+	And the Send CV form is clear
+	Example: https://career.quantori.com/ru/positions/cloud-python-engineer
```
Comment (yankovskayaksenia) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:7]: No example in Background

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+
+	
+@Type:Positive
+@HappyPass
+@Priority:Critical
+Scenario Outline: 3.1_01 Fill the Send CV form form with correct data
+	
+	When user enters correct data into the field "Name": <Name>
```
Comment (yankovskayaksenia) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:15]: You should provide specific data what should be put in the fields

File: qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+	And user enters correct data into the field "Email": <Email>
+	And user presses "Attach CV" button and choses a file in correct format (PDF / DOC / DOCX / RTF / Max 10 Mb)
```
Comment (yankovskayaksenia) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature:17]: several actions should be placed on different lines
